docs/cloud-native-security/kspm.asciidoc

```diff
@@ -1,6 +1,81 @@
 [[kspm]]
 = Kubernetes security posture management
```

Comment:
Overall this topic feels kind of unusual compared to most of our other docs. Typically section overview topics like this don't actually contain a lot of procedures; they're just trying to give a general summary of the feature set. Reproducing the procedures from other topics can also be a real challenge for future updates and maintenance. 

I think this is fine to proceed with for now, but it'd be good to keep iterating on the whole section after release, and maybe refactor to make maintenance easier.

Reply:
It would be very good @benironside if we still could add file
https://github.com/elastic/security-docs/blob/main/docs/cloud-native-security/get-started-with-kspm.asciidoc
to this pr and link the integrations strings `Kubernetes Security Posture Management` to the appropriate integrations documentation https://docs.elastic.co/integrations/cloud_security_posture

Reply:
Good summary here @joepeeples, I agree it's a lot of repetition with other topics. I think @tinnytintin10's intention here is to make the information about use-cases as accessible as possible, but there's definitely a trade-off with how readable and maintainable the whole section is (repetition is detrimental on both counts). I'm supportive of trimming it down. Curious to hear your thoughts @jmikell821

Reply:
@111andre111 sorry, I'm not understanding this comment. 

What do you mean by "add file https://github.com/elastic/security-docs/blob/main/docs/cloud-native-security/get-started-with-kspm.asciidoc"? Are you suggesting that we link from this page to the "Get started with KSPM" page?

What do you mean by "integrations strings"? I think you are suggesting we link from this page to https://docs.elastic.co/integrations/cloud_security_posture (which makes sense to me) but I'm not sure.

Reply:
Sorry if I have not been clear enough.
I what I wanted to say here is if it is possible to link on these 2 lines:
https://github.com/elastic/security-docs/blame/main/docs/cloud-native-security/get-started-with-kspm.asciidoc#L27
https://github.com/elastic/security-docs/blame/main/docs/cloud-native-security/get-started-with-kspm.asciidoc#L208

String `Kubernetes Security Posture Management` to https://docs.elastic.co/integrations/cloud_security_posture

Does that mke it more clear, @benironside ?

Reply:
I like Andre's suggestion. @benironside I know that [this integration docs view](https://docs.elastic.co/integrations/cloud_security_posture) is in technical preview; however, if you're okay linking out to it, I full support it.

Reply:
@joepeeples one thing our product struggles with is helping users get value out of our many features. There is much work around PLG to help in this space; however, it's still not near delivery. I view detailed use case-based documentation as part of the product and a solution to helping users understand the value they can extract from our product and certain features. 

With this in mind, I wanted to make this section as clear and detailed as possible by describing what KSPM is and giving examples of how users can extract value. I will rely on y'all to trim the content down in all the right places, but I do want to retain its guiding and descriptive nature in regard to what the product is and how users can gain value.